.github/workflows/verify.yaml

```diff
@@ -312,6 +312,30 @@ jobs:
     outputs:
       has-azure: ${{ steps.has-azure.outputs.defined }}
 
+  Kafka-Integration-Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup-build
+
+      - name: 'Upgrade docker-compose (for --wait option)'
```

Comment:
The problem is that I do need a default network to be also created (which docker-compose does automatically).
I have no idea it that works with github services (did not find that in the doc)

Reply:
as far as i know, GH Actions assumes that all services are "depended on", so it will wait for all declared services to be up-and-ready, using their `HEALTCHCHECK` if they have it. At least that is my experience with it. 

If you want to go forward with docker-compose, I would suggest using a `condition`, if zookeeper supports it: 

```yaml
 depends_on:
      zookeeper:
        condition: service_healthy 
```

Reply:
Removed `docker-compose` and replaced with GitHub services ;)